person.rb

```diff
@@ -1,17 +1,32 @@
-class Person
+require './nameable'
+require './capitilize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] You are required to create `getters & setters` for the `:name, :age, :rentals` so using `attr_accessor` would be appropriate in this instance for the `rentals`, Please kindly add this to your code and resubmit for approval :+1: :sparkles: